setup.py

```diff
@@ -24,6 +24,9 @@ def get_version(rel_path):
     version=get_version('azure_acquire/__init__.py'),
     packages=find_packages(),
     platforms=['mac', 'unix'],
+    install_requires=['click', 'Sphinx', 'coverage', 'flake8', 'python-dotenv>=0.5.1',
+                      'opencv-python', 'numpy', 'matplotlib', 'jupyter', 'pathlib2', 
+                      'tqdm', 'pyk4a', 'jupyter'],
```

Comment:
most of these packages are unnecessary for running the acquisition code. Remove all packages that aren't directly called by `cli.py` and `util.py`